docker/Dockerfile

```diff
@@ -8,6 +8,16 @@
 # - release: builds release binary
 # - runtime: is our runtime environment
 #
+# We first set default values for build arguments used across the stages.
+# Each stage must define the build arguments (ARGs) it uses.
+#
+# Build zebrad with these features
+# Keep these in sync with:
+# https://github.com/ZcashFoundation/zebra/blob/main/.github/workflows/build-docker-image.yml#L42
+ARG FEATURES="default-release-binaries"
+ARG TEST_FEATURES="${FEATURES} lightwalletd-grpc-tests zebra-checkpoints"
+ARG NETWORK="Mainnet"
```

Comment:
###### Optional

Wouldn't it be clearer to have the default values set in each stage that uses the build arg? We need to redeclare these args later on anyway, and if there are declared new ones in a given stage, it might be hard to tell if they have a default value without looking up here.